ingester/ingester.py

```diff
@@ -60,8 +61,30 @@ def move_file_to_failed_dir(filename, failed_dir):
         print(f"Error moving file {filename} to failed directory: {e}")
         raise e
 
+TREES_FILE = "/app/trees.yml"
 
-def ingest_submissions(spool_dir, db_client=None):
+def load_trees_name():
+    """Load trees.yml into a dict: {url -> tree_name}."""
```

Comment:
I'm not sure you saw the rest of my comment, since you've converted it to a docstring, but it's still not reflective of the code